dask_expr/_describe.py

```diff
@@ -52,21 +52,25 @@ def _lower(self):
             frame.max(split_every=self.split_every),
         ]
         return DescribeNumericAggregate(
-            self.frame._meta.name, is_td_col, is_dt_col, *stats
+            self.frame._meta.name,
+            is_td_col,
+            is_dt_col,
+            getattr(self.frame._meta.array, "unit", None),
+            *stats,
         )
 
 
 class DescribeNumericAggregate(Blockwise):
-    _parameters = ["name", "is_timedelta_col", "is_datetime_col"]
+    _parameters = ["name", "is_timedelta_col", "is_datetime_col", "unit"]
     _defaults = {"is_timedelta_col": False, "is_datetime_col": False}
 
     def _broadcast_dep(self, dep):
         return dep.npartitions == 1
 
     @staticmethod
-    def operation(name, is_timedelta_col, is_datetime_col, *stats):
+    def operation(name, is_timedelta_col, is_datetime_col, unit, *stats):
         return describe_numeric_aggregate(
-            stats, name, is_timedelta_col, is_datetime_col
+            stats, name, is_timedelta_col, is_datetime_col, unit
         )
```

Comment:
As in #1110, this new argument to a `dask` function means you need to depend on `dask>=2024.8.0` now.

Reply:
We updated our pinning to `dask == 2024.8.0` in the release earlier today https://github.com/dask/dask-expr/commit/837d44c56c97d463aa12cdcae47c626a458413f8